### PR TITLE
Several improvements

### PR DESCRIPTION
**Pull request by adamkaplan**
- Fulfill requirement to not delete with 5+ members and recent bot
  messages
- Send admin message to Slack in Dry Run mode for debugging
- Send only 1 admin message, no spam on first run (I got 950 spams!)
- Add whitelist.txt support, 1 exclusion rule per line
- Support for POST requests (Admin messages can get too long for GET)

### CODE REVIEW

File: slack-autoarchive.py [hunk 1]
```diff
@@ -13,18 +13,22 @@
 ADMIN_CHANNEL      = os.getenv('ADMIN_CHANNEL')
 AUDIT_LOG          = 'audit.log'
 DAYS_INACTIVE      = 60
+MIN_MEMBERS        = 5
```
Comment (JohnTheodore) [inline slack-autoarchive.py]: I want to archive channels even if there are arbitrary users in there, 50, 500, etc. Would you modify your PR so MIN_MEMBERS is unlimited by default, and you can set say an environment variable to limit archiving to only channels below 5 members to restrict archiving to small channels?

**Review by JohnTheodore — CHANGES_REQUESTED** [review]
Thoughts on setting the default member limit to unlimited, and opting into a limit?

File: slack-autoarchive.py (hunk 1 above)
Comment (adamkaplan) [reply slack-autoarchive.py]: Sure, sounds fine.  I only chose 5 due to the README. (However, so far 5 has proved to work well for my 1700 channel Slack instance!)

**Review by JohnTheodore — APPROVED** [review]

**Comment by JohnTheodore** [comment]
LGTM

**Pull request merged**